tests/controllers/EventHostsControllerTest.php

```diff
@@ -2,18 +2,16 @@
 
 namespace JoindinTest\Controller;
 
-use Mockery as M;
 use PHPUnit\Framework\TestCase;
 
 class EventHostsControllerTest extends TestCase
 {
-    /**
-     * @expectedException \Exception
```

Comment:
annotation-level exception assertions are deprecated in PHPUnit 8. https://github.com/sebastianbergmann/phpunit/issues/3332